spring-cloud-gcp-autoconfigure/src/test/java/com/google/cloud/spring/autoconfigure/kms/KmsAutoConfigurationTests.java

```diff
@@ -18,33 +18,43 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 
 import com.google.api.gax.core.CredentialsProvider;
 import com.google.auth.Credentials;
+import com.google.auth.oauth2.ServiceAccountCredentials;
+import com.google.auth.oauth2.UserCredentials;
 import com.google.cloud.kms.v1.KeyManagementServiceClient;
+import com.google.cloud.spring.core.GcpProjectIdProvider;
 import com.google.cloud.spring.kms.KmsTemplate;
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.WebApplicationType;
+import org.springframework.boot.autoconfigure.AutoConfigurations;
 import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.boot.test.context.runner.ApplicationContextRunner;
 import org.springframework.context.ConfigurableApplicationContext;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
-/** Unit tests for {@link GcpKmsAutoConfiguration}. */
+/**
+ * Unit tests for {@link GcpKmsAutoConfiguration}.
+ */
 class KmsAutoConfigurationTests {
 
-  private static final String PROJECT_NAME = "hollow-light-of-the-sealed-land";
-  private static final String LOCATION_NAME = "global";
-  private static final String KEY_RING_NAME = "key-ring-id";
-  private static final String KEY_ID_NAME = "key-id";
+  private static final String CORE_PROJECT_NAME = "core-project";
+  private static final String KMS_PROJECT_NAME = "hollow-light-of-the-sealed-land";
+  private static final String KMS_CREDENTIAL_LOCATION = "src/test/resources/fake-credential-key.json";
+  private static final String CORE_CREDENTIAL_CLIENT_ID = "12345";
+  private static final String KMS_CREDENTIAL_CLIENT_ID = "45678";
+
+  private SpringApplicationBuilder applicationBuilder = new SpringApplicationBuilder(
+      TestConfiguration.class, GcpKmsAutoConfiguration.class).properties(
+      "spring.cloud.gcp.kms.project-id=" + KMS_PROJECT_NAME, "spring.cloud.bootstrap.enabled=true",
+      "spring.cloud.gcp.sql.enabled=false").web(WebApplicationType.NONE);
```

Comment:
original formatting with property per line was probably easier to read.

Reply:
Ah! thanks for catching this. My formatter messed up in a few places, reverting this.